tests/test_views.py

```diff
@@ -0,0 +1,750 @@
+import json
+
+from lxml import objectify
+import pytest
+
+from django.core.urlresolvers import reverse
+from django.http import Http404
+from django.utils import timezone
+
+from premis_event_service import views, models
+from . import factories, AppEventTestXml
+
+
+pytestmark = [
+    pytest.mark.urls('premis_event_service.urls'),
+    pytest.mark.django_db,
+]
+
+
+def test_app_returns_ok(rf):
+    request = rf.get('/')
+    response = views.app(request)
+    assert response.status_code == 200
+
+
+def test_app_response_content_type(rf):
+    request = rf.get('/')
+    response = views.app(request)
+    assert response.get('Content-Type') == 'application/atom+xml'
+
+
+def test_humanEvent_raises_http404(rf):
+    """An Http404 should be raised when the view is unable to locate
+    an Event.
+    """
+    request = rf.get('/')
+    with pytest.raises(Http404):
+        views.humanEvent(request)
+
+
+def test_humanEvent_returns_ok(rf):
+    event = factories.EventFactory.create()
+    request = rf.get('/')
+    response = views.humanEvent(request, event.event_identifier)
+    assert response.status_code == 200
+
+
+def test_json_event_list_returns_ok(rf):
```

Comment:
What do you think about renaming this and the next couple tests to `test_recent_event_list_*` rather than `test_json_event_list_*`?